doc/sphinx/source/graph-basics.rst

```diff
@@ -1,25 +1,93 @@
 .. graph-basics:
 
+#############
 Graph Basics
-============
+#############
 
-This section describes the basic concepts you need to know when 
-constructing a graph.
 
+This section explains some of the primary concepts used in the nGraph Library.
+Some new ideas regarding our unique departure from the first generation of deep 
+learning software design are also introduced. 
 
-Framework Bridges
-------------------
+Many frameworks have a baked-in assumption that performant deep learning systems 
+require stacks of GPUs. Given this assumption, the frameworks tend to yield best 
+performance `only when given full control`_ of the GPU devices, and if 
+`any other resource demands`_ are made on the GPU, the GPU ends up performing about 
+the same as a CPU. Indeed, many GPU manufacturers have gone out of their way to 
+boast of multi-GPU to CPU performance, which is essentially an apples-to-oranges 
+comparison for this very reason. 
 
-Frontends (or users who require the flexibility of constructing 
-Ops directly) can utilize a set of graph construction functions 
-to construct graphs. 
+As different kinds of deep learning applications emerge, and as layers of 
+complexity for anything other than a :abbr:`Just-in-Time (JIT)` compilation are 
+explored, limitations inherent in the first generation of DNNs become more apparent. 
+For example, most GPU frameworks that expect full control of the GPU devices 
+experience their own per-device inefficiency for resource utilization 
+whenever the system encounters a bottleneck. In other words, if a deep learning 
+model requires any operation that hasn't been implemented on GPU, it must wait 
+for copies to propagate to the GPUs. Most framework owners running GPUs 
+will tell you to refactor model to remove the unimplemented copy, rather than 
+attempt to run multiple models in parallel. Frameworks that compile with the 
+nGraph library, however, can be prescriptively programmed to work around such 
+limitations and can save the data scientist the pain of having to refactor 
+their model.    
 
-A framework bridge constructs a function which is compiled/optimized
-by a sequence of graph transformations that replace subgraphs of the
-computation with more optimal subgraphs. Throughout this process, ops
-represent tensor operations.
+While the first generation frameworks tended to need to make a tradeoff between 
+being "specialized" and "adaptable" (the trade-off between training and inference), 
+nGraph allows algorithms implemented in a DNN to be both specialized and adaptable. 
+The new generation of software design in and around AI ecosystems can and 
+should be much more flexible.   
 
 
+* :ref:`framework_bridges`
+* :ref:`about_transformers`
+* :ref:`graph_shaping`
+ 
+
+
+.. _framework_bridges:
+
+Framework bridges
+=================
+
+In the nGraph ecosystem, a framework is any "frontend" used by a data 
```

Comment:
s/frontend/software tool/ or something like that. ("Frontend" is kind of compiler jargon-y, I think.)

Reply:
Done